rasa/utils/train_utils.py

```diff
@@ -1275,3 +1275,16 @@ def load_tensor(name: Text) -> Optional[Union["tf.Tensor", List["tf.Tensor"]]]:
         return tensor_list[0]
 
     return tensor_list
+
+
+def normalize(values: "np.ndarray", ranking_length: Optional[int] = 0) -> None:
+    """Normalizes an array of positive numbers over the top `ranking_length` values, if provided.
+
+    Other values will be set to 0. Happens in place."""
+
+    if 0 < ranking_length < len(values):
+        ranked = sorted(values, reverse=True)
+        values[values < ranked[ranking_length - 1]] = 0
+
+    if np.sum(values) > 0:
+        values /= np.sum(values)
```

Comment:
```suggestion
        values = values / np.sum(values)
        
    return values
```

Reply:
I tried in Jupyter notebook, we need return values. Original array will not be mutated

Reply:
I'm confused, aren't lists mutable? it's the same thought process of having list.reverse() that returns nothing. What did you try in jupyter that didn't work?

Reply:
values that you pass to function will not be mutated. List is different. In general it is not recommended to mutate input of a function